include/mbedtls/ssl.h

```diff
@@ -42,7 +42,7 @@
 #include "mbedtls/dhm.h"
 #endif
 
-#if defined(MBEDTLS_ECDH_C)
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_ECDH)
```

Comment:
No: this is non-PSA code so it needs to keep referencing `MBEDTLS_ECDH_C`. Maybe `|| MBEDTLS_PSA_…`, depending on whether `ecdh.h` is needed when `MBEDTLS_USE_PSA_CRYPTO` is enabled, which I haven't checked.

Likewise in `ssl_internal.h` and `ssl_*.c`.

Reply:
I'll back these changes out.